README.rst

```diff
@@ -40,6 +40,23 @@ Remember, if co-installing multiple CWL implementations then you need to
 maintain which implementation ``cwl-runner`` points to via a symbolic file
 system link or `another facility <https://wiki.debian.org/DebianAlternatives>`_.
 
+Running tests locally
+---------------------
+
+-  Running basic tests ``(/tests)``:
+
+.. code:: bash
+
+    python setup.py test
+
+-  Running the entire suite of CWL conformance tests:
+
+The GitHub repostiory for the CWL specifications contains a script that tests a CWL
+implementation against a wide array of valid CWL files using the [cwltest](https://github.com/common-workflow-language/cwltest)
```

Comment:
you're using markdown formatting for inserting hyperlinks. I can add a commit to fix it for rst formatting.